src/dealDashboard/dealDescription/dealDescription.ts

```diff
@@ -1,9 +1,42 @@
-import { autoinject } from "aurelia-framework";
+import { autoinject, computedFrom } from "aurelia-framework";
 import { bindable } from "aurelia-typed-observable-plugin";
 import { DealTokenSwap } from "entities/DealTokenSwap";
+import { MobileService } from "services/MobileService";
 import "./dealDescription.scss";
 
+const MOBILE_MAX_LENGTH = 250;
+
 @autoinject
 export class DealDescription {
   @bindable deal: DealTokenSwap;
+
+  /**
+   * Description text is shortened to 250 words
+   */
+  private descriptionText: string | undefined;
+  private originalDescriptionText = "";
+
+  @computedFrom("descriptionText")
+  get showReadMoreButton() {
+    return this.descriptionText.length < this.originalDescriptionText.length;
+  }
+
+  bind() {
+    this.originalDescriptionText = this.deal.registrationData.proposal.description;
+
+    if (MobileService.isMobile()) {
```

Comment:
Currently, when one resizes the browser (eg. from 375px to +376px), then nothing will change.

@dkent600 Should I also add a viewport listener?